client/components/marketing-survey/cancel-purchase-form/step-components/downgrade-step.jsx

```diff
@@ -68,6 +79,42 @@ export class DowngradeStep extends Component {
 						{ args: { amount } }
 				  );
 		}
+
+		if ( 'downgrade-monthly' === upsell ) {
+			refundTitle = translate(
+				'Would you rather switch to the more affordable monthly subscription?'
+			);
+
+			if ( cancelBundledDomain && includedDomainPurchase ) {
+				refundReason = (
+					<p>
+						{ translate(
+							'You will keep all features of your current plan including the domain %(domain)s but the plan period will be reduced.',
```

Comment:
Probably it should say "most of the features" here (like below).  I think they still lose things like live chat?